### PR TITLE
update vgg: input range [0, 1], docs, rst, remove redundant files

### PR DESCRIPTION
**Pull request by warshallrho**
### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

update vgg input from [0, 255]-mean to [0, 1] (both in static and dynamic modes), update related docs, rst, and remove redundant files

### CODE REVIEW

File: docs/user/get_start_advance.rst [hunk 1]
```diff
@@ -13,20 +13,42 @@ Get entire CNN
...
-
+  # get VGG without the last layer
+  cnn = tl.models.vgg16(end_with='fc2_relu', mode='static').as_layer()
+  # add one more layer and build a new model
+  ni = Input([None, 224, 224, 3], name="inputs")
+  nn = cnn(ni)
+  nn = tl.layers.Dense(n_units=100, name='out')(nn)
+  model = tl.models.Model(inputs=ni, outputs=nn)
+  # train your own classifier (only update the last layer)
+  train_params = model.get_layer('out').weights
```
Comment (ChrisWu1997) [inline docs/user/get_start_advance.rst]: I think this cannot work for VGG, as we encapsulated all layers into a LayerList for VGG, unlike MobileNet or SqueezeNet.

Comment (ChrisWu1997) [reply docs/user/get_start_advance.rst]: Oh sorry, I misread this line. This should work.

**Comment by ChrisWu1997** [comment]
Could you also modify the `Get specific weights` section in `get_start_model.rst`. We does not support `get_weights` now, better to use `get_layer` to first get specific layer and then `.weights`.

**Comment by warshallrho** [comment]
> Could you also modify the `Get specific weights` section in `get_start_model.rst`. We does not support `get_weights` now, better to use `get_layer` to first get specific layer and then `.weights`.

I have updated it

**Pull request merged**